changelogs/v249/skip_database_provision.md

```diff
@@ -0,0 +1,15 @@
+## Apps that use the `heroku/ruby` buildpack can now skip database provisioning on the first push with the  `HEROKU_SKIP_DATABASE_PROVISION` environment variable.
+
+When applications are deployed to Heroku, the last buildpack to execute can request add-ons are provisioned via the [`bin/release` interface](https://devcenter.heroku.com/articles/buildpack-api#bin-release). The `heroku/ruby` buildpack checks if the application [contains a Postgres gem](https://github.com/heroku/heroku-buildpack-ruby/blob/104fe3a374e07a8f3723f110c2148d57ecb9ee79/lib/language_pack/ruby.rb#L992-L1008) and request that Heroku provision a database for the application automatically. Before the pricing change, the database requested was free, and adding this add-on by the buildpack would save developers time while setting up applications.
```

Comment:
Oh also, when posting the changelog entry, the URL linking to the buildpack source for the implementation will need updating to point to the new SHA after merge, otherwise users following the link will see a file that doesn't implement the env var check, and might get confused :-)